sr_robot_commander/doc/tutorial/HandCommander.rst

```diff
@@ -1,172 +1,80 @@
 Hand Commander
---------------
+-------------
 
-Description
+Overview
 ~~~~~~~~~~~
 
-This commander provides commands specific to hand which allows to
-execute all actions of the `robot commander <RobotCommander.md>`__. Also
-it allows to get state of tactile sensors, set max force and get joints
-effort.
+The SrHandCommander inherits all methods from the `robot commander <RobotCommander.html>`__ and provides commands specific to the hand. It allows to get state of tactile sensors, set maximum force and get joints effort.
```

Comment:
'It allows the state of the tactile sensors to be read, the maximum force to be set and the joints effort to be read'